### PR TITLE
SUBMARINE-939. Fix Auto Reload  Checkbox Not To Continuously Reset

### PR DESCRIPTION
**Pull request by noidname01**
### What is this PR for?

Fix Auto Reload  Checkbox Not To Continuously Reset

### What type of PR is it?
[Bug Fix]

### Todos

None

### What is the Jira issue?

https://issues.apache.org/jira/projects/SUBMARINE/issues/SUBMARINE-939

### How should this be tested?

You can create an experiment and check the checkbox, then see whether the checkbox is not reload together with other components.

### Screenshots (if appropriate)

https://user-images.githubusercontent.com/55401762/125742649-f57b4c3c-4de4-4721-9982-a2cd8110875d.mp4

### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? No

### CODE REVIEW

**Comment by noidname01** [comment]
@ByronHsu I fix the bug you mentioned~

File: submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-home.component.ts [hunk 1]
```diff
@@ -76,7 +76,7 @@ export class ExperimentHomeComponent implements OnInit {
     this.onSwitchAutoReload();
   }
 
-  fetchExperimentList() {
+  fetchExperimentList(isRefetch: boolean) {
```
Comment (ByronHsu) [inline submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-home.component.ts:79]: I think `isAutoReload` is easier to understand.

File: submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-home.component.ts [hunk 2]
```diff
@@ -94,9 +94,11 @@ export class ExperimentHomeComponent implements OnInit {
             item.duration = this.experimentService.durationHandle(result);
           }
         });
-        this.checkedList = [];
-        for (let i = 0; i < this.experimentList.length; i++) {
-          this.checkedList.push(false);
+        if(!isRefetch){
```
Comment (ByronHsu) [inline submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-home.component.ts:97]: Add a comment here
// If it is auto-reloading, we do not want to change the state of checkbox.

File: submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-home.component.ts (hunk 1 above)
Comment (noidname01) [reply submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-home.component.ts:79]: Oh, You're right. I'll modify it ASAP.

File: submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-home.component.ts (hunk 2 above)
Comment (noidname01) [reply submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-home.component.ts:97]: Got it.

File: submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-home.component.ts (hunk 1 above)
Comment (ByronHsu) [reply submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-home.component.ts:79]: Oh you replied so fast xd

**Comment by ByronHsu** [comment]
LGTM

**Review by pingsutw — APPROVED** [review]
LGTM

**Pull request closed without merging**